openprescribing/templates/_practice_list.html

```diff
@@ -0,0 +1,16 @@
+<p>
+  There are {{ num_open_practices }} practices currently active in this CCG
```

Comment:
I think you need to replace `CCG` with `{% entity_type_human %}`, or whatever the variable is called, so this works with PCNs.

Reply:
Ach, I had that in the first commit.  Well spotted.